docs/source/configuration.rst

```diff
@@ -88,6 +88,37 @@ in projects which utilise a lot of complicated templating.
 
 .. _templateconfig:
 
+Rule Configuration
+------------------
+
+Rules can be configured with the `.sqlfluff` config files.
+
+Common rule configurations can be set in the *[sqlfluff:rules]* section.
+
+For example:
+
+.. code-block:: cfg
+
+    [sqlfluff:rules]
+    unquoted_identifiers_policy = all
+    max_line_length = 88
+
+Rule specific configurations are set in the *[sqlfluff:rules:\*]* sections.
```

Comment:
I find that syntax a little confusing to read. What about this:

```suggestion
Rule specific configurations are set in subsections under each rule id.
```

Reply:
Is it just me or is it confusing to se that the subsections are "under" rule ids? Would this work: "..set in rule specific subsections"?

Reply:
Yup that works for me.